scale/storage/management/commands/scale_destroy_file.py

```diff
@@ -0,0 +1,85 @@
+"""Defines the command line method for running a destroy file task"""
+from __future__ import unicode_literals
+
+import ast
+import json
+import logging
+import signal
+import sys
+from collections import namedtuple
+
+from django.core.management.base import BaseCommand
+
+import storage.destroy_files_job as destroy_files_job
+from storage.brokers.factory import get_broker
+from storage.configuration.workspace_configuration import WorkspaceConfiguration
+from messaging.manager import CommandMessageManager
+from storage import destroy_files_job
+from storage.messages.delete_files import create_delete_files_messages
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Command that executes the destroy file process for a given file
+    """
+
+    help = 'Perform a file destruction operation on a file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('-f', '--files', nargs='+', type=json.loads, required=True,
+                            help='File path and ID in a JSON string.' +
+                            ' e.g: "{"file_path":"some.file", "id":"399"}"')
+        parser.add_argument('-j', '--job_id', action='store', type=int, required=True,
+                            help='ID of the Job model')
+        parser.add_argument('-w', '--workspace', action='store', type=json.loads, required=True,
+                            help='Workspace configuration in a JSON string.')
+        parser.add_argument('-p', '--purge', action='store', type=bool, required=True,
+                            help='Purge all records for the given file')
```

Comment:
I would recommend using env vars for the inputs instead of the command line. There could be any number of workspaces that house the files, so you will need to be able to specify multiple workspaces and indicate which workspace each file belongs to. I would also recommend trying a strategy like this to break up the inputs:
NUM_FILES = 5
FILE_1_ID = 1
FILE_1_PATH = relative/workspace/path/file.txt
FILE_1_WORKSPACE = workspace_1
...
FILE_5_ID = 5
FILE_5_PATH = relative/workspace/path/file_5.txt
FILE_5_WORKSPACE = workspace_2
NUM_WORKSPACES = 2
WORKSPACE_1_NAME = workspace_1
WORKSPACE_1_BROKER = {"json": "config"}
WORKSPACE_2_NAME = workspace_2
WORKSPACE_2_BROKER = {"json": "config"}